Doc/library/asyncio-task.rst

```diff
@@ -624,32 +624,28 @@ Timeouts
     The context manager produced by :func:`asyncio.timeout` can be
     rescheduled to a different deadline and inspected.
 
-    .. class:: Timeout()
+    .. class:: Timeout(when)
 
        An :ref:`asynchronous context manager <async-context-managers>`
-       that limits time spent inside of it.
+       for cancelling overdue coroutines.
 
-        .. versionadded:: 3.11
+       ``when`` should be either a float relative to the current
```

Comment:
“Relative” is really confusing here, since it is an “absolute” time, just using the time scale of loop.time(). (I don’t know what the official term is, what do we say for reschedule() or timeout_at()?)

Reply:
`reschedule` just says "change the time" (though we're changing that as part of this PR). `timeout_at` uses the term "absolute time": https://docs.python.org/3/library/asyncio-task.html#asyncio.timeout_at

Reply:
How about this phrasing:

"``when`` should be an absolute time at which the context should time out,
       as measured by the event loop's clock."

Reply:
+1

Reply:
SGTM too!

Reply:
Done